README.md

```diff
@@ -94,6 +94,32 @@ ResponseDumper.define 'Users' do
 end
 ```
 
+## Before/After Hooks
```

Comment:
Up to you, IMO, this should go above "reset_models" as the feature seems a bit more fundamental and common.